ipatests/test_ipapython/test_ssh.py

```diff
@@ -69,6 +69,8 @@
      u'%s this is a comment' % openssh),
     (u'opt3,opt2="\tx ",opt1,opt2="\\"x " %s comment ' % openssh,
      u'opt1,opt2="\\"x ",opt3 %s comment' % openssh),
+    (u'permitopen=\"1.1.1.1:111\",permitopen=\"2.2.2.2:222\" %s' % openssh,
+     u'permitopen=\"1.1.1.1:111\",permitopen=\"2.2.2.2:222\" %s' % openssh),
```

Comment:
Did you mean to use `permitlisten` in the second line?

Reply:
No, that line is the expected output for the test. Before the change, only one `permitopen` entry would be returned. I guess I could add another test input/output for `permitlisten`, but it works exactly the same as `permitopen`.

Reply:
Ok, understood.

Please add the `permitlisten` to make it clear we expect it working this way for both commands.